.buildkite/pipeline.yml

```diff
@@ -1,4 +1,86 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+env:
+  SETUP_GVM_VERSION: "v0.5.1"
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
+  DOCKER_REGISTRY: "docker.elastic.co"
+  DOCKER_IMG: "${DOCKER_REGISTRY}/observability/stream"
+  DOCKER_IMG_PR: "${DOCKER_REGISTRY}/observability-ci/stream"
+
 steps:
-  - label: "Test"
-    command: "echo Test me!"
+  - label: ":golangci-lint: Lint"
+    key: "lint"
+    command:
+      - "sudo apt-get update"
+      - "sudo apt-get install -y libpcap-dev"
```

Comment:
For unattended installations with `apt-get` the best practice is to define the environment variables:

```
export DEBIAN_FRONTEND=noninteractive
export DEBIAN_PRIORITY=critical
```

see https://manpages.ubuntu.com/manpages/focal/en/man7/debconf.7.html for more details.

The same comment applies to the remaining steps, so these env vars could be set globally at the pipeline level too I think.